bookkeeper-server/src/main/java/org/apache/bookkeeper/replication/Auditor.java

```diff
@@ -347,6 +348,7 @@ public Auditor(final String bookieIdentifier,
         this.numLedgersFoundHavingLessThanAQReplicasOfAnEntry = new AtomicInteger(0);
         this.numLedgersHavingLessThanWQReplicasOfAnEntryGuageValue = new AtomicInteger(0);
         this.numLedgersFoundHavingLessThanWQReplicasOfAnEntry = new AtomicInteger(0);
+        this.semaphore = new Semaphore(conf.getAuditorGetLedgerSemaphore());
```

Comment:
maybe it is better to validate this value (must be greater than zero) and throw an exception with a useful error message (and indicate the invalid config prop name)